pkg/apis/elasticsearch/v1/elasticsearch_types.go

```diff
@@ -175,6 +175,11 @@ func (r RemoteCluster) ConfigHash() string {
 	return hash.HashObject(r)
 }
 
+func (es Elasticsearch) IsRemoteClustersDefined() bool {
+	return len(es.Spec.RemoteClusters) > 0 &&
+		es.Spec.RemoteClusters[0].Name != "" && es.Spec.RemoteClusters[0].ElasticsearchRef.IsDefined()
```

Comment:
Should we iterate over all remote clusters in case the first one is not correctly defined?

Reply:
Yes, but I ended up reuse `getRemoteClustersInSpec()`.